### PR TITLE
[FIX] 1696 & 1267: 【レイアウト崩れ】ゲスト購入、ご注文内容確認のお客様情報編集フォーム

### PR DESCRIPTION
**Pull request by lqdung-lockon**
1. Fix issue 1696 & 1267
2. Add unit test.

### CODE REVIEW

**Comment by hoand-vn** [comment]
tested + reviewed : Okie

File: src/Eccube/Controller/ShoppingController.php [hunk 1]
```diff
@@ -789,86 +791,96 @@ public function shippingEdit(Application $app, Request $request, $id)
 
     /**
      * お客様情報の変更(非会員)
+     *
+     * @param Application $app
+     * @param Request     $request
      */
     public function customer(Application $app, Request $request)
```
Comment (t-nagahashi) [inline src/Eccube/Controller/ShoppingController.php]: Why did you  refactor this function?
I think that It is difficult to read ....

File: src/Eccube/Controller/ShoppingController.php [hunk 2]
```diff
@@ -1410,7 +1422,20 @@ private function customerValidation(Application $app, array $data)
         $errors[] = $app['validator']->validateValue($data['customer_name02'], array(
             new Assert\NotBlank(),
             new Assert\Length(array('max' => $app['config']['name_len'],)),
-            new Assert\Regex(array('pattern' => '/^[^\s ]+$/u', 'message' => 'form.type.name.firstname.nothasspace'))
+            new Assert\Regex(array('pattern' => '/^[^\s ]+$/u', 'message' => 'form.type.name.lastname.nothasspace'))
+        ));
+        $data['customer_kana01'] = Converter::convertToKana($data['customer_kana01']);
```
Comment (t-nagahashi) [inline src/Eccube/Controller/ShoppingController.php]: I think this soursecode is OK.

before fixing: `Converter::convertToKana($data['customer_kana01']);`
my solution: `mb_convert_kana($data['customer_kana01'], 'CV', 'utf-8');`

File: src/Eccube/Resource/template/default/Shopping/index.twig [hunk 3]
```diff
@@ -58,8 +57,12 @@ $(function() {
         var name = [];
         var input = [];
         var customerin = [];
+        var customerEditButton = $('#customer');
```
Comment (t-nagahashi) [inline src/Eccube/Resource/template/default/Shopping/index.twig]: when you use "jquery variable", you should put `$`.

`customerEditButton`  -> `$customerEditButton`

**Pull request closed without merging**